src/store/slices/users/authService.ts

```diff
@@ -14,15 +16,17 @@ export const signInUser = createAsyncThunk(
     };
     await signInWithEmailAndPassword(auth, data.email, data.password)
       .then((userCredential) => {
+        const splitName = userCredential.user.displayName?.split(' ');
         userData.loginData = {
           email: userCredential.user.email,
           uid: userCredential.user.uid,
-          displayName: userCredential.user.displayName,
           photoURL: userCredential.user.photoURL,
           phoneNumber: userCredential.user.phoneNumber,
           emailVerified: userCredential.user.emailVerified,
-          isAnonymous: userCredential.user.isAnonymous,
-          metadata: userCredential.user.metadata
+          firstName: splitName ? splitName[0] : '',
```

Comment:
De asta ai uitat?

Reply:
Da, dar am verificat acum si daca le scot vin erori de type-uri pe ele. Ideea este ca la google displayName vine direct ca string nu ca "string || null" si de asta nu este necesara sintaxa.

Asa ca propunerea mea este sa il lasama sa momentan pana ne mai dumirim cu firebase si sa facem alte componente mai importante.

Reply:
De fapt stai ca acolo este o incurcatura pentru ca este functia de log in nu de logare.  Revin cu commit dupa ce modific cum trebuie.